### DISCUSSION TITLE
How do I install and use this?

### DISCUSSION THREAD

**Discussion by CodedNil** [bjbr-dev/easy-gesture#11] [Q&A]
I updated my kde plasma to the new 5.25 and now easystroke broke, so I would like to move to this new program since easystroke is abandoned, how do I get it working?
I ran the build.sh and then have tried running the easy-gestured that created but can't get anything running.

**Comment by revast** [comment]
You don't.
Its unmaintained.
But there is a modern replacement, [https://github.com/jersou/mouse-actions](https://github.com/jersou/mouse-actions), which works on both X11 and Wayland.
